### PR TITLE
Fix bugs in indexed dataset exam script

### PR DESCRIPTION
**Pull request by yidong72**
Fixed some bugs in exam ecript

### CODE REVIEW

File: nemo/collections/nlp/data/language_modeling/megatron/indexed_retrieval_dataset.py [hunk 1]
```diff
@@ -430,7 +430,7 @@ def get_chunk(self, chunk_id, force_no_padding=False):
         """
         if isinstance(chunk_id, (int, np.int64, np.int32)):
             ptr = self._index.get_chunk_address(chunk_id)
-            if self._index.retrieval_db:
+            if self._index.retrieval_db and (not force_no_padding):
```
Comment (michalivne) [inline nemo/collections/nlp/data/language_modeling/megatron/indexed_retrieval_dataset.py:433]: Can you change name `force_no_padding` to be more indicative of function?  Maybe `force_no_cont_text`?

**Review by michalivne — COMMENTED** [review]
Minor comment.

**Review by michalivne — COMMENTED** [review]
LGTM. Minor comment.

File: nemo/collections/nlp/data/language_modeling/megatron/indexed_retrieval_dataset.py (hunk 1 above)
Comment (yidong72) [reply nemo/collections/nlp/data/language_modeling/megatron/indexed_retrieval_dataset.py]: changed the name to `force_no_cont_ids`

**Review by michalivne — APPROVED** [review]
LGTM!

**Pull request merged**